conf/jira.yaml.template

```diff
@@ -3,3 +3,6 @@ JIRA:
   URL: https://issues.redhat.com
   # Provide api_key to access Jira REST API
   API_KEY: replace-with-jira-api-key
+  COMMENT_TYPE: comment_type
+  COMMENT_VISIBILITY: private_or_public
+  ENABLE_COMMENT: false
```

Comment:
Remove this setting as commented earlier, keeping this enabled unintentionally in user system will add comments to jira until noticed.

Keep only option.

Reply:
@jyejare No, it won't add any comment unless both pytest option and conf are set. That's what I mentioned in the PR description too. This is to avoid accidental use.

Reply:
See https://github.com/SatelliteQE/robottelo/pull/15104/files#diff-3550e1a2f365bdaca4a1639200efd36e262592bc3b48a61c29131cbd5df1b274R51 and https://github.com/SatelliteQE/robottelo/pull/15104/files#diff-5a1a1d6684ec1cb9a1cfe346439a95b30eb2272e3a0671fc0810e57ee447c241R295 and

Reply:
@jameerpathan111 Got you, but it's not really required to have same option in both places and that too both should be turned on for the purpose.

Reply:
I think it's better this way but I can keep only pytest option if everyone feels the same. @JacobCallahan @ogajduse @omkarkhatavkar thoughts?